### PR TITLE
created tax crud

### PR DESCRIPTION
**Pull request by BaltasRokis**

### CODE REVIEW

File: Looms.PoS/Looms.PoS.Application/Features/Tax/Commands/CreateTax/CreateTaxRequestValidator.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+public class CreateTaxRequestValidator : AbstractValidator<CreateTaxRequest>
+{
+    public CreateTaxRequestValidator()
+    {
+        RuleFor(x => x.Percentage)
+            .Cascade(CascadeMode.Stop)
+            .NotEmpty()
+            .InclusiveBetween(0, 100)
+            .WithMessage("Percentage must be between 0 and 100.");
```
Comment (titasstank) [inline Looms.PoS/Looms.PoS.Application/Features/Tax/Commands/CreateTax/CreateTaxRequestValidator.cs:16]: Not needed

File: Looms.PoS/Looms.PoS.Application/Features/Tax/Commands/UpdateTax/UpdateTaxCommandHandler.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
+using Looms.PoS.Application.Interfaces;
+using Looms.PoS.Application.Interfaces.ModelsResolvers;
+using Looms.PoS.Application.Models.Requests.Tax;
+using Looms.PoS.Domain.Interfaces;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Looms.PoS.Application.Features.Tax.Commands.UpdateTax;
+public class UpdateTaxCommandHandler : IRequestHandler<UpdateTaxCommand, IActionResult>
```
Comment (titasstank) [inline Looms.PoS/Looms.PoS.Application/Features/Tax/Commands/UpdateTax/UpdateTaxCommandHandler.cs:9]: new line after namespace

File: Looms.PoS/Looms.PoS.Application/Features/Tax/Commands/UpdateTax/UpdateTaxRequestValidator.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
+using FluentValidation;
+using Looms.PoS.Application.Utilities.Validators;
+using Looms.PoS.Application.Models.Requests.Tax;
+using Looms.PoS.Domain.Enums;
+
+namespace Looms.PoS.Application.Features.Tax.Commands.UpdateTax;
+
+//TODO
```
Comment (titasstank) [inline Looms.PoS/Looms.PoS.Application/Features/Tax/Commands/UpdateTax/UpdateTaxRequestValidator.cs:8]: Remove the comment or write what to do

File: Looms.PoS/Looms.PoS.Application/Mappings/Profiles/TaxProfile.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+using Looms.PoS.Application.Models.Responses.Tax;
+using Looms.PoS.Application.Utilities;
+using Looms.PoS.Domain.Daos;
+using Looms.PoS.Domain.Enums;
+
+namespace Looms.PoS.Application.Mappings.Profiles;
+
+public class TaxProfile : Profile
+{
+    //TODO
```
Comment (titasstank) [inline Looms.PoS/Looms.PoS.Application/Mappings/Profiles/TaxProfile.cs:12]: Remove or fix

File: Looms.PoS/Looms.PoS.Application/Models/Responses/Tax/TaxResponse.cs [hunk 5]
```diff
@@ -0,0 +1,14 @@
+using Looms.PoS.Domain.Enums;
+
+namespace Looms.PoS.Application.Models.Responses.Tax;
+
+public class TaxResponse
```
Comment (titasstank) [inline Looms.PoS/Looms.PoS.Application/Models/Responses/Tax/TaxResponse.cs:5]: record

**Review by titasstank — APPROVED** [review]
Fix comments and can merge

File: Looms.PoS/Looms.PoS.Application/Features/Tax/Commands/UpdateTax/UpdateTaxCommandHandler.cs (hunk 2 above)
Comment (BaltasRokis) [reply Looms.PoS/Looms.PoS.Application/Features/Tax/Commands/UpdateTax/UpdateTaxCommandHandler.cs:10]: added

File: Looms.PoS/Looms.PoS.Application/Features/Tax/Commands/CreateTax/CreateTaxRequestValidator.cs (hunk 1 above)
Comment (BaltasRokis) [reply Looms.PoS/Looms.PoS.Application/Features/Tax/Commands/CreateTax/CreateTaxRequestValidator.cs]: deleted

File: Looms.PoS/Looms.PoS.Application/Mappings/Profiles/TaxProfile.cs (hunk 4 above)
Comment (BaltasRokis) [reply Looms.PoS/Looms.PoS.Application/Mappings/Profiles/TaxProfile.cs]: removed

File: Looms.PoS/Looms.PoS.Application/Models/Responses/Tax/TaxResponse.cs (hunk 5 above)
Comment (titasstank) [reply Looms.PoS/Looms.PoS.Application/Models/Responses/Tax/TaxResponse.cs:5]: no

Comment (BaltasRokis) [reply Looms.PoS/Looms.PoS.Application/Models/Responses/Tax/TaxResponse.cs:5]: but like every other response is class?

Comment (titasstank) [reply Looms.PoS/Looms.PoS.Application/Models/Responses/Tax/TaxResponse.cs:5]: then every response should be changed to record :D

Comment (BaltasRokis) [reply Looms.PoS/Looms.PoS.Application/Models/Responses/Tax/TaxResponse.cs]: changed it to record

**Review by titasstank — APPROVED** [review]

**Pull request merged**